models/newsModel.js

```diff
@@ -75,7 +75,53 @@ exports.getAllUsers = () => {
   const queryString = `SELECT * FROM users`;
 
   return db.query(queryString).then((results) => {
-    console.log(results.rows);
     return results.rows;
   });
 };
+
+exports.getQueriedArticles = (topic, sort_by, order) => {
```

Comment:
Could we add some default values for topic and sort_by if the values are undefined please?